### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by remcohaszing**
This saves about 60KB bundle size (unminified).

### CODE REVIEW

File: build.js [hunk 1]
```diff
@@ -4,13 +4,14 @@ import { fileURLToPath } from 'url';
 
 import { build } from 'esbuild';
 
+const [, , logLevel = 'info'] = process.argv;
```
Comment (remcohaszing) [inline build.js:7]: Not really related, but it’s useful for troubleshooting the local built.

```sh
npm run prepack debug
```

File: build.js [hunk 2]
```diff
@@ -78,6 +79,14 @@ await build({
         // None of our dependencies use side effects, but many packages don’t explicitly define
         // this.
         onResolve({ filter: /.*/ }, () => ({ sideEffects: false }));
+
+        // Rewrite the tailwind stubs from CJS to ESM, so our bundle doesn’t need to include any CJS
+        // related logic.
+        onLoad({ filter: /\/tailwindcss\/stubs\/defaultConfig\.stub\.js$/ }, async ({ path }) => {
+          const cjs = await readFile(path, 'utf8');
+          const esm = cjs.replace('module.exports =', 'export default');
```
Comment (mhsdesign) [inline build.js:87]: 😄

File: build.js [hunk 3]
```diff
@@ -22,13 +23,13 @@ await build({
...
       name: 'alias',
-      async setup({ onResolve, resolve }) {
+      async setup({ onLoad, onResolve, resolve }) {
         const stubFiles = await readdir('src/stubs', { withFileTypes: true });
         // These packages are imported, but can be stubbed.
         const stubNames = stubFiles
           .filter((file) => file.isFile())
           .map((file) => parse(file.name).name);
-        onResolve({ filter: new RegExp(`^(${stubNames.join('|')})$`) }, ({ path }) => ({
+        onResolve({ filter: new RegExp(`^(${stubNames.join('|')})(\\/|$)`) }, ({ path }) => ({
```
Comment (mhsdesign) [inline build.js:32]: when `utils` would be a stubname,
not only
`import "utils"`
and
`import "utils/"`
but also
`import "utils/foo/bar/bung/baz"`
would be matched. is this wanted?
dont you mean `/^(utils)\/?$/`

Comment (remcohaszing) [reply build.js]: You’re right. This shouldn’t have changed. I reverted it.

**Pull request merged**